src/CommunityToolkit.Maui.UnitTests/Converters/IntToBoolConverter_Tests.cs

```diff
@@ -13,9 +13,11 @@ public void IndexToArrayConverter(int value, bool expectedResult)
 	{
 		var intToBoolConverter = new IntToBoolConverter();
 
-		var result = (bool)intToBoolConverter.Convert(value, typeof(bool), null, CultureInfo.CurrentCulture);
+		var result = intToBoolConverter.Convert(value, typeof(bool), null, CultureInfo.CurrentCulture);
+		var typedResult = intToBoolConverter.ConvertFrom(value);
 
 		Assert.Equal(result, expectedResult);
+		Assert.Equal(typedResult, expectedResult);
```

Comment:
Let's swap these `Assert.Equal` parameters around.

It's kinda trivial, but when the `Assert` fails, it'll print "...Expected: [first parameter] Actual: [second parameter]" which can be confusing if the parameters are reversed

```suggestion
		Assert.Equal(expectedResult, result);
		Assert.Equal(expectedResult, typedResult);
```